docs/content/operator/validator-committee.mdx

```diff
@@ -1,5 +1,8 @@
 ---
 title: Validator Committee
+tags: [node-operation]
```

Comment:
I disagree that this is tag "node-operation".
This page is about basic concepts explanation, but not about operating a node.
And for the team tag, maybe consensus fits more here? What do you think @piotrm50 ?

Reply:
Maybe we should even move the whole page out of the "Node operators" section?
Does it maybe fit better in "About IOTA" => "IOTA Architecture" ?